src/tool_getparam.c

```diff
@@ -1370,13 +1370,13 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
          */
         const char *p = strchr(nextarg, '=');
         size_t nlen;
-        char is_file;
+        size_t is_file;
```

Comment:
Why? It reads a single character from a buffer into the variable

Reply:
Agreed, I'll change it back. What made me go this direction was this line `nlen = is_file = 0;` made my linter complain, because obviously `0` isn't a `char` (but `'\0'` is). Regardless this is a common C-ism, so I'll revert it.